config-reloader/templates/kubernetes.conf

```diff
@@ -10,7 +10,6 @@
   read_from_head true
   <parse>
     @type json
-    time_format %Y-%m-%dT%H:%M:%S.%NZ
```

Comment:
Hi, I have tested it with stdout as well as elastic-search out plugin and it worked fine without loosing any logs. docker daemon adds the timestamp with every log with the key as "time" and it is in json format, so every log line has the time. By default the parser directive deletes the timestamp anyway, that is the case here too. So, irrespective of whether we are parsing the time or not it gets deleted from the log message once it comes out of the in_tail plugin. To preserve the timestamp, we need to add `keep_time_key true` under `<parse>` directive.

I will reconfirm it with fluentd guys and inquire about the performance too.

Reply:
@apu1111 did you confirm with fluentd about this?

Reply:
Hi @carpenterm 
I have checked with fluentd guys and looks like it's not a problem. You can check more details here: https://github.com/fluent/fluentd/issues/2504. I have asked other relevant questions but not yet answered.
So, if you would like to merge this PR, please remove the above line and proceed. Let me know if you need any help.
I might work on the https://github.com/vmware/kube-fluentd-operator/issues/60, if it is still relevant. Let me know.

Reply:
@kkapoor1987, Thanks for bringing these points. We have not observed any performance issues on our environments. Can you try building the fluentd-operator with the changes suggested by @apu1111 and see the improvements? This will hep us to test and merge the changes\

Reply:
@dimalbaby I am in middle of testing it today and confirm the results.

Reply:
@dimalbaby So I spend some time didnt really see much improvement when I applied these changes. Plus some f the changes change how log streams are created in CLoudWatch. 
Any recommendations to increase the performance of fluentd ?

Reply:
@kkapoor1987 We have seen some performance issue with due a plugin https://github.com/vmware/kube-fluentd-operator/issues/121. The issue had been fixed. I am not sure whether you have tested with the same plugin version. Can you try with the beta build https://github.com/vmware/kube-fluentd-operator/releases/tag/v1.13.0-beta.2 or the changes from master + the suggested changes above ?